create_gz_vendor_pkg/templates/CMakeLists.txt.jinja

```diff
@@ -16,6 +16,17 @@ set(LIB_NAME_COMP_PREFIX ${LIB_NAME})
 set(LIB_NAME_FULL ${LIB_NAME}${LIB_VER_MAJOR})
 set(LIB_VER ${LIB_VER_MAJOR}.${LIB_VER_MINOR}.${LIB_VER_PATCH})
 
+option(VENDOR_FROM_LIB_VCS_REF
+  "Vendor from a VCS ref instead of tag. Uses the value in LIB_VCS_REF if specified or main otherwise"
+  OFF)
+if(NOT VENDOR_FROM_LIB_VCS_REF)
+  set(LIB_VCS_VER ${GITHUB_NAME}${LIB_VER_MAJOR}_${LIB_VER}${LIB_VER_SUFFIX})
+elseif(LIB_VCS_REF)
+  set(LIB_VCS_VER ${LIB_VCS_REF})
+else()
+  set(LIB_VCS_VER main)
```

Comment:
Do we need a special case for gz-tools here?

Reply:
I added some logic in https://github.com/gazebo-tooling/gz_vendor/pull/16/commits/8b933051c7b8b05f35b19ff0021050aa91840300 but I haven't tested it since I've never run the script before